src/app/settings/panel-settings/components/panel-input-modifiers/panel-input-modifiers.component.ts

```diff
@@ -37,4 +49,19 @@ export class PanelInputModifiersComponent implements OnInit {
       this.panelInputModifierStateService.createPanelInputModifier(panelInputModifier)
     })
   }
+
+  public deletePanelInputModifier(panelInputModifier: PanelInputModifier): void {
+    this.dialogService.createConfirmDialog(
```

Comment:
```suggestion
if(isDeleteDisabled(panelInputModifier)) {
  return
}

    this.dialogService.createConfirmDialog(
```

Pseudo code to make it so that the button doesn't react if you press it :)

Reply:
This method has nothing to do with the button and has no influence on the button.

Sidenote: The buttons already don't react if they are disabled.